game/NonExportedEnums.h

```diff
@@ -10,7 +10,7 @@
 
 
 
-
+[[deprecated("AllPlayers will be removed in a future release of Outpost2DLL, Use enum PlayerNum::PlayerALL instead")]]
```

Comment:
Capitalization of "All" does not match code.

Reply:
Thanks for noticing.